.github/workflows/branch-tests.yml

```diff
@@ -47,4 +48,4 @@ jobs:
         run: python -c "from river import datasets; datasets.CreditCard().download(); datasets.Elec2().download()"
 
       - name: pytest
-        run: pytest
+        run: pytest -n logical # Run pytest on all logical CPU cores
```

Comment:
I'm assuming `-n logical` provides the fastest results?

Reply:
It should — yes. I just did not want to hard code this in case the runner specs changed in the future.

In the existing GitHub runners, `-n auto` and `-n logical` should give the same result, though.